ghedesigner/radial_numerical_borehole.py

```diff
@@ -41,40 +41,38 @@ def __init__(self, single_u_tube: SingleUTube):
         self.num_grout_cells = 27
         self.num_soil_cells = 500
 
-        self.num_cells = self.num_fluid_cells + self.num_conv_cells + self.num_fluid_cells
-        self.num_cells += self.num_grout_cells + self.num_soil_cells + 1
+        self.num_cells = self.num_fluid_cells + self.num_conv_cells + self.num_pipe_cells
+        self.num_cells += self.num_grout_cells + self.num_soil_cells
```

Comment:
This was a bug, but it had no effect. I looked at the "+1" and thought, "hmm, this is odd, why do we need that?" It's because `num_fluid_cells` (which has a value of 3) was counted twice, and `num_pipe_cells` (which has a value of 4) wasn't counted.